package.json

```diff
@@ -7,14 +7,16 @@
     "postinstall": "husky install && preconstruct dev && manypkg check && check-node-version --package --print",
     "reinstall": "manypkg exec rm -rf node_modules && rm -rf node_modules pnpm-lock.yaml && pnpm install",
     "auth": "npm_config_registry=https://registry.npmjs.org npm whoami",
-    "develop": "jest --projects jest.*.config.js --watch",
-    "lint": "jest --config jest.lint.config.js --maxWorkers 5",
+    "develop": "jest --projects jest.test.config.js --watch",
+    "lint": "biome lint",
     "typecheck": "tsc --project tsconfig.json --noEmit && tsc --project packages/cypress-plugin/tsconfig.json --noEmit",
     "format": "pnpm format:ts && pnpm format:md && pnpm format:yaml",
-    "format:ts": "prettier --write '**/*.{js, ts, tsx}'",
+    "format:ts": "biome format --write",
     "format:md": "prettier --parser markdown --write '**/*.md'",
     "format:yaml": "prettier --parser yaml --write '**/*.{yml,yaml}'",
```

Comment:
As mentioned above biome language support is quite limited so yaml and md files are not supported yet which is why I kept prettier for that.